### PR TITLE
Add dev application identity for `vc-exchanger`.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Comment by mattcollier** [comment]
@JSAssassin can you explain why this is necessary?  It certainly may be the case, but usually the default app identity key is used in dev:

https://github.com/digitalbazaar/bedrock-app-identity/blob/main/lib/config.js#L9

**Comment by JSAssassin** [comment]
@mattcollier without this service ID configuration, I get the following error when trying to generate an exchanger config in `veres-exchanger`.  Noticed that the same is done for the vc issuer and verifier, so I thought this must the right thing to do. 
https://github.com/digitalbazaar/veres-meter/blob/7cb8e47a039961976c8314c26e355eb5ec7ab9d5/configs/veres-meter.js#L27-L45
https://github.com/digitalbazaar/bedrock-vc-issuer/blob/main/lib/config.js#L39-L46
https://github.com/digitalbazaar/bedrock-vc-verifier/blob/main/lib/config.js#L39-L46
```
2023-07-19T13:44:07.748Z - error: [express] error workerPid=70875, workerId=491dd71d86b2f249, details={
  "error": {
    "name": "NotAllowedError",
    "message": "Expected meter service type \"vc-exchanger\" is not a configured client.",
    "details": {
      "url": "https://localhost:59443/meters/z19rFxRY87zW8qmBr8d8sAQYx/usage",
      "httpStatusCode": 403,
      "public": true
    },
    "cause": null,
    "stack": "NotAllowedError: Expected meter service type \"vc-exchanger\" is not a configured client.\n    at _getUsage (file:///Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/@bedrock/meter-usage-reporter/lib/meters.js:492:11)\n    at Module.hasAvailable (file:///Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/@bedrock/meter-usage-reporter/lib/meters.js:234:47)\n    at file:///Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/@bedrock/service-core/lib/http/configs.js:48:50\n    at asyncMiddleware (file:///Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/@bedrock/express/lib/index.js:562:20)\n    at Layer.handle [as handle_request] (/Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/express/lib/router/route.js:144:13)\n    at validate (file:///Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/@bedrock/validation/lib/index.js:192:5)\n    at Layer.handle [as handle_request] (/Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/express/lib/router/route.js:144:13)\n    at cors (/Users/tashid.gyeltshen/Desktop/Veres/veres-exchanger/node_modules/cors/lib/index.js:188:7)",
    "type": "NotAllowedError"
  }
}
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # bedrock-vc-delivery ChangeLog
 
+## 3.1.0 - 2023-07-TBD
```
Comment (JSAssassin) [inline CHANGELOG.md:3]: Should I mark this as a breaking change? Given that the deployments will need to include a similar configuration.

Comment (dlongley) [reply CHANGELOG.md]: I don't think this needs to be breaking; it was a mistake not to have it and it didn't function without it. So there was already a requirement for it.

Comment (mattcollier) [reply CHANGELOG.md]: Also, it's already been setup for deployments: https://github.com/digitalbazaar/terraform-gcp-veres-exchanger/blob/main/config.tftpl#L7

**Pull request merged**